src/pykeen/utils.py

```diff
@@ -440,3 +441,20 @@ def ensure_ftp_directory(*, ftp: ftplib.FTP, directory: str) -> None:
         ftp.mkd(directory)
     except ftplib.error_perm:
         pass  # its fine...
+
+
+def invert_mapping(mapping: Mapping[str, int]) -> Mapping[int, str]:
+    """
+    Invert a mapping.
+
+    :param mapping:
+        The mapping, key -> value.
+
+    :return:
+        The inverse mapping, value -> key.
+    """
+    num_unique_values = len(set(mapping.values()))
+    num_keys = len(mapping)
+    if num_unique_values < num_keys:
+        raise ValueError(f'Mapping is not bijective! Only {num_unique_values}/{num_keys} are unique.')
+    return dict(map(reversed, mapping.items()))
```

Comment:
maybe `{v:k for k,v in mapping.items()}` would get the point across a bit easier